packages/fxa-event-broker/README.md

```diff
@@ -1,3 +1,27 @@
 # fxa-event-broker
 
-TODO: write stuff
+## Deployment
+
+### Testing a Webhook URL
+
+To verify that a webhookUrl can process a payload correctly, the simulator can
+be used to generate a simulated JWT signed payload. The relying party can verify
+this worked correctly before a full deployment occurs.
+
+Command Syntax:
+
+    node dist/bin/simulate-webhook-call.js CLIENTID WEBHOOKURL CAPABILITIES
+
+Where `CAPABILITIES` is a comma-seperated string of capabilities to include.
+
+#### Usage
+
+```bash
+
+    $ node run build
+    $ export LOG_FORMAT=pretty
+    $ node dist/bin/simulate-webhook-call.js a9238ba https://example.com/webhook capability_1
```

Comment:
@bbangert you mentioned you use httpbin for testing. could you paste your working snippet in here?

Reply:
Sure, using say, `https://requestbin.fullcontact.com/1mhpma41?inspect`:

```bash
$ export OPENID_KEYFILE=$REPO_PATH/packages/fxa-auth-server/config/secret-key.json
$ node dist/bin/simulate-webhook-call.js a9238ba https://requestbin.fullcontact.com/1mhpma41 test_capability
```